Documentation/Settings.md

```diff
@@ -255,7 +261,7 @@ Loop icon animations in main menu
 
 ### Setting: Anim. speed
 
-How fast should the menu animations loop, or if they should not loop at all.
+How fast should the menu animations loop or off setting.
```

Comment:
Please leave it as it is.
Since the sentence you propose doesn't make any sense.